products.json

```
@@ -1,33 +1,44 @@
 {
     "androidstudio": {
+        "display-name": "Android Studio",
         "folder-name": "AndroidStudio",
-        "preferences-path": "~/Library/Application Support/Google/"
+        "preferences-path": "~/Library/Application Support/Google/",
+        "bundle-id": "com.google.android.studio"
     },
     "appcode": {
-        "folder-name": "AppCode"
+        "folder-name": "AppCode",
+        "bundle-id": "com.jetbrains.appcode"
     },
     "clion": {
-        "folder-name": "CLion"
+        "folder-name": "CLion",
+        "bundle-id": "com.jetbrains.clion"
     },
     "datagrip": {
-        "folder-name": "DataGrip"
+        "folder-name": "DataGrip",
+        "bundle-id": "com.jetbrains.datagrip"
     },
     "goland": {
-        "folder-name": "GoLand"
+        "folder-name": "GoLand",
+        "bundle-id": "com.jetbrains.goland"
     },
     "idea": {
-        "folder-name": "IntelliJIdea"
+        "folder-name": "IntelliJIdea",
+        "bundle-id": "com.jetbrains.idea"
```

this has to be `intellij` (see https://github.com/artemy/alfred-jetbrains-projects/blob/4e85d16c85986d30e2775c4edb4a0fb466f6b47d/build.py#L9)
```suggestion
        "bundle-id": "com.jetbrains.intellij"
```